use_sso_authentication/index.rst

```diff
@@ -0,0 +1,106 @@
+
+You will have to configure the Service Provider information with the following data:
+
+* EntityId: http://localhost:8080/saml/metadata
+* Logout url: http://localhost:8080/saml/logout
+* ACS url: http://localhost:8080/saml/acs
+
+You will have to retrieve the IDP certificate, it will be needed to configure the Service Provider on the Akeneo PIM side.
+
+
+Setting up the Service Provider (SP)
+------------------------------------
+
+* Go to the `sso configuration page`, under ``System/SAML-based Single Sign-On``
+* Enable the feature using the "SSO Enable" toggle button
+* Fill the form with the values provided by your IdP server
+* Then click the Save button
+* If everything went fine, you'll be logged out and redirected to the IDP login form
+
+For the IDP provided in the docker-compose.yml
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Entity ID: ``http://localhost:8081/simplesaml/saml2/idp/metadata.php``
+* Login URL: ``http://localhost:8081/simplesaml/saml2/idp/SSOService.php``
+* Logout URL: ``http://localhost:8081/simplesaml/saml2/idp/SingleLogoutService.php``
+* Certificate: ``MIIDXTCCAkWgAwIBAgIJALmVVuDWu4NYMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwHhcNMTYxMjMxMTQzNDQ3WhcNNDgwNjI1MTQzNDQ3WjBFMQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzUCFozgNb1h1M0jzNRSCjhOBnR+uVbVpaWfXYIR+AhWDdEe5ryY+CgavOg8bfLybyzFdehlYdDRgkedEB/GjG8aJw06l0qF4jDOAw0kEygWCu2mcH7XOxRt+YAH3TVHa/Hu1W3WjzkobqqqLQ8gkKWWM27fOgAZ6GieaJBN6VBSMMcPey3HWLBmc+TYJmv1dbaO2jHhKh8pfKw0W12VM8P1PIO8gv4Phu/uuJYieBWKixBEyy0lHjyixYFCR12xdh4CA47q958ZRGnnDUGFVE1QhgRacJCOZ9bd5t9mr8KLaVBYTCJo5ERE8jymab5dPqe5qKfJsCZiqWglbjUo9twIDAQABo1AwTjAdBgNVHQ4EFgQUxpuwcs/CYQOyui+r1G+3KxBNhxkwHwYDVR0jBBgwFoAUxpuwcs/CYQOyui+r1G+3KxBNhxkwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAAiWUKs/2x/viNCKi3Y6blEuCtAGhzOOZ9EjrvJ8+COH3Rag3tVBWrcBZ3/uhhPq5gy9lqw4OkvEws99/5jFsX1FJ6MKBgqfuy7yh5s1YfM0ANHYczMmYpZeAcQf2CGAaVfwTTfSlzNLsF2lW/ly7yapFzlYSJLGoVE+OHEu8g5SlNACUEfkXw+5Eghh+KzlIN7R6Q7r2ixWNFBC/jWf7NKUfJyX8qIG5md1YUeT6GBW9Bm2/1/RiO24JTaYlfLdKK9TYb8sG5B+OLab2DImG99CJ25RkAcSobWNF5zD0O6lgOo3cEdB/ksCq3hmtlC/DlLZ/D8CJ+7VuZnS1rR2naQ==``
+
+
+FAQ
+---
+
+What if i've wrongly configured my admin user, i can't access to the PIM anymore ?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An original login form is here as a fallback in this situation, you can access it here: http://your.akeneo-pim.url/user/login
+
+How can i debug authentication errors ?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Various errors can happen, those errors could be due to incorrect configuration on IdP side or SP side.
+
+Regarding the SP, on the sso configuration page, you have a "download logs" that will provide a ZIP of ``var/logs/authentication*.log`` or check thoses logs manually via the CLI.
```

Comment:
Splitting the sentence in two is more readable imo : "You can also check these logs via CLI if you PIM installation is on premise or on PaaS"